editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -7337,9 +7343,9 @@ void Node3DEditor::_init_grid() {
 		grid_mat[c]->set_shader_parameter("orthogonal", orthogonal);
 
 		// Cache these so we don't have to re-access memory.
```

Comment:
Not sure this comment is accurate, these seem just aliases.
Maybe it's trying to refer to pre-reserving memory instead of `push_back()`.

Reply:
Its saying we are caching the reference to the array so we don't have to do the array access. I.e. we can use `ref_grid` instead of `grid_points[c]` everywhere. I'm not sure this is worth a comment.

Reply:
Ah I get it if it is trying to avoid the array access, the comment could be better here. :+1: 
I would (hope) this would normally get compiled out in release, but maybe not in debug.

Reply:
Yeah I agree the comment could just be removed.

Reply:
Will do.

Reply:
Done.